Setup.md

```diff
@@ -0,0 +1,76 @@
+* Fimrware **MUST** be updated prior to anything
+* Adding an additional source for the board manager
+* Configure the Arduino *alike* processor
+* Choose an appropriate *.ino file
+
+# Firmware
+The firmware is an essential part and need to be flashed before any other operation. The good news it is doable directly from your webbrowser.
+
+**Prerequesites**
+* An actual Browser such like Chrome
+* Web Serial Access allowes (Web Serial API)
+* All-in-One PCB is connected to your PC
+
+After prereuisites are fulfilled follow the instructions here [Tonuino All-in-One online firmware updater](https://tonuino.de/aio_update.html)
+
+# Arduino IDE
+The main thing is to add the *Leiterkartenpiraten* processor settings to the Arduino IDE. Therefore you need to add an additional source for the board manager. 
+
+## Add Additional Borad Manager Source
+
+#|Action|Result
+---|---|---
+1|**Add additional source**|
+1.1| Naviagte to *Datei>Voreinstellungen* | The preferences view is shown.
+2.1| Select text box *Zusätzliche Bordverwalter-URLs* and add the the follwing|https://raw.githubusercontent.com/dbuezas/lgt8fx/master/package_lgt8fx_index.json| Additional source is shown in the text box
+3.1| Confirm your changes with *OK*| The preferences view closes and the main window of Arduino IDE is shown|
+
+## Select appropriate board and architecture
+
+**Please note: Now there are two (or more) menu items in *Werkzeuge>Boards* This is caused by adding the additional URL and is intended**
+
+#|Action|Result
+---|---|---
+1|**Select board**|
+1.1|Navigate to *Werkzeuge>Boards>Logic Green Arduino AVR Compatible Boards*| Associated boards are shown such like LT...
+1.2|Select *LGT8F328*| The board is selected and the menu closes
+2|**Select Clock Source** |
+2.1|Navigate to *Werkzeuge>Clock Source:*| The associated clock sources are shown (Internal, External,...)
+2.2|Select *External: (assumes 32Mhz crystal)*| The Clock source is selected and the menu closes|
+3|**Select Clock**|
+3.1|Navigate to *Werkzeuge>Clock*| The associated clock speeds are shown|
+3.2|Select *16Mhz*| The clock speed is selected and the menu closes|
+4|**Select Variant**|
+4.1|Naviagte to *Werkzeuge>Variant:*|The associated variants are shown|
+4.2|Select *328P-LQFP32 (e.g. MiniEVB nano-style and WAVGAT)*| The Variant is selected and the menu closes|
+5|**Select Arduino as ISP**|
+5.1|Navigate to *Werkzeuge>Arduino as ISP*| The associated architectures are shown|
+5.2|Select *Default (64)*| The architecture is selected and the menu closes
+
+## Set Monitor Baudrate
+As this is all serial, the baudrate is essentitial to set appropriately. For the All-in-One PCB it is *115200*
+
+**Please Note: Setting the Baudrate to any other that 115200 Baud the monitor will display a satans dialect directly from hell (hiroglyphs)**
```

Comment:
**Please Note: Setting the baud rate to any other value than 115200 Baud will result in unreadable output**